### DISCUSSION TITLE
does `System.Drawing.Image` convert to `Eto.Drawing.Image`?

### DISCUSSION THREAD

**Discussion by visualizor** [picoe/Eto#2001] [Q&A]
any faster way than converting the system image to byte[] and construct new eto image?

p.s. i have PNGs loaded in visual studio as resources and I noticed the `Eto.Drawing.Bitmap.FromResource()`, but I don't know how to correctly point to the png

**Comment by cwensley** [comment]
Set the image as an Embedded Resource in its properties and use its full namespace with `FromResource()`.  Using .resx files is not entirely cross platform.

E.g. if you have a project named "MyProject" and you put the embedded resource in "Resources\MyFile.png", then you would use `Eto.Drawing.Bitmap.FromResource("MyProject.Resources.MyFile.png")`

**Reply by visualizor** [reply]
![image](https://user-images.githubusercontent.com/25107288/127072942-6ddd46ff-76bd-47ab-ba1b-66265d6d2c8f.png)

![image](https://user-images.githubusercontent.com/25107288/127073283-6366604e-8bf3-4eee-9c87-d8dde808b559.png)

i'm puzzled...
there is no embed in the properties panel, either
![image](https://user-images.githubusercontent.com/25107288/127073385-66ec84a5-9b02-4880-9c89-ae8a446127dd.png)

**Reply by cwensley** [reply]
For `Build Action`, set it to `Embedded Resource` instead of `None`.
